packages/api/schema.gql

```diff
@@ -173,29 +159,18 @@ type PaginatedEmailCampaigns {
   totalCount: Int!
 }
 
-type BrevoConfig implements DocumentInterface {
-  id: ID!
-  updatedAt: DateTime!
-  senderMail: String!
-  senderName: String!
-  createdAt: DateTime!
-  scope: EmailCampaignContentScope!
-}
-
 input EmailCampaignContentScopeInput {
   thisScopeHasNoFields____: String
 }
 
 type Query {
-  brevoContact(id: Int!): BrevoContact!
+  brevoContact(id: Int!, scope: EmailCampaignContentScopeInput!): BrevoContact!
```

Comment:
Is it necessary to pass the scope here? Shouldn't the ID be unique across scopes?

Reply:
The problem here is that the contact is not stored in the database with a scope, but simply in brevo without any knowledge which scope the contact belongs too. The contacts of a specific scope are stored inside a "main" target list for the scope and the target group information is stored in the database. The scope cannot easily be stored within the brevo contacts (contact can be within more that one list for example and brevo limits additional attributes to really basic datatypes only) - that's why the target groups is used for storing the scope information. And since a contact can be in multiple scopes the scope cannot be retrieved that way.